Promact.Oauth.Server/src/Promact.Oauth.Server/Consent/ConsentOptions.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace Promact.Oauth.Server.Consent
+{
+    public class ConsentOptions
+    {
+        public static bool EnableOfflineAccess = true;
+        public static string OfflineAccessDisplayName = "Offline Access";
```

Comment:
All strings should be in string constants. We don't need EnableOfflineAccess so, this option can be set to false or true wherever it is required